less/core/nav.less

```diff
@@ -1,16 +1,27 @@
 // Add top padding equal to the height of the navigation bar
 // --------------------------------------------------
-#wrapper {
+html:not(.is-nav-bottom) #wrapper {
```

Comment:
Code seems to suggest only top / bottom nav available (we could expand to left / right in the future). Would suggest each navigation position gets its own class and padding is added depending on which nav is active rather than not active.